dev/src/document.ts

```diff
@@ -971,29 +961,19 @@ export class DocumentTransform<T = DocumentData> {
   }
 
   /**
-   * Converts a document transform to the Firestore 'DocumentTransform' Proto.
+   * Converts a document transform to the Firestore 'FieldTransform' Proto.
    *
    * @private
    * @param serializer The Firestore serializer
-   * @returns A Firestore 'DocumentTransform' Proto or 'null' if this transform
-   * is empty.
+   * @returns A list of Firestore 'FieldTransform' Protos
    */
-  toProto(serializer: Serializer): api.IWrite | null {
-    if (this.isEmpty) {
-      return null;
-    }
-
-    const fieldTransforms: api.DocumentTransform.IFieldTransform[] = [];
+  toProto(serializer: Serializer): api.DocumentTransform.IFieldTransform[] {
+    const fieldTransforms = [];
     for (const [path, transform] of this.transforms) {
       fieldTransforms.push(transform.toProto(serializer, path));
     }
 
-    return {
-      transform: {
-        document: this.ref.formattedName,
-        fieldTransforms,
-      },
-    };
+    return fieldTransforms;
```

Comment:
Optional: Use `map` to turn this function into one line.

Reply:
attempted, but used `Array.from()`